.github/CONTRIBUTING.md

```diff
@@ -41,6 +41,139 @@ Our procedures for contributing bigger changes, code in particular, generally fo
 - We use [testthat](https://cran.r-project.org/package=testthat) for unit tests.
   Contributions with test cases included are easier to accept.
 
+## Working with a development version of `hubverse-org/schemas`
+
+The canonical home for the hubverse schemas are at
+https://github.com/hubverse-org/schemas. These schemas are copied over here
+under the `inst/schemas` folder, which allows offline validation for hubs.
+
+**If you are developing against an in-development version of the hubverse
+schemas, you must ensure that the schemas in this repository are synchronized**
+
+### Synchronization script
+
+The script that synchronizes the schemas is in
+[data-raw/schemas.R](https://github.com/hubverse-org/hubUtils/blob/main/data-raw/schemas.R)
+and it can be run from within R, as a standalone script, or as a git hook. It
+takes one environment variable `HUBUTILS_DEV_BRANCH`. If the environment
+variable is unset, the branch information from the `inst/schemas/update.json`
+is used.
+
+#### Usage: within R
+
+```r
+source("data-raw/schemas.R")
+```
+
+#### Usage: from BASH
+
+```bash
+Rscript data-raw/schemas.R
+```
+
+#### Usage: commit hook
+
+See [Installing the Git Hook](#installing-the-git-hook). A Git hook is a way to
+run a local script before or after you do something in Git. For example, a
+pre-commit hook (the one we use here) will run every time before you make a
+commit. Likewise, a pre-push hook will run every time before you push to a
+repository.
```

Comment:
NOTE: I chose a pre-commit hook as the entry point over pre-push, but it really could be either one. A pre-commit is nice because it's a way to constantly update, but at the cost of an API call per commit. A pre-push is nice because you are only going to push when you know you have internet, but the downside is that if it synchronizes, you have to make a new commit and push.

Reply:
scratch that, I've moved to using a `pre-push` hook since that uses less resources.